jenkins/maven-kubectl/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and limitations under the
+# License.
+FROM maven:3.6-jdk-8
```

Comment:
My one concern is that a majority of the build time is spent downloading maven dependencies. Is it possible to include the pom file in this dockerfile, and pre-download the dependencies before creating it? This would increase the frequency of changes to the image (everytime a dependency is changed) but might be worthwhile with the time and network resources saved for subsequent builds.

Reply:
I think the correct solution here is installing artifactory as a service within the cluster so that we have a local cache. I'll have to do something investigating as to the proper approach for configuring that.